services/app-web/src/pages/SubmissionRevisionSummary/SubmissionRevisionSummary.module.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../styles/uswdsImports.scss';
+@import '../../styles/custom.scss';
+
+.background {
+    background-color: $cms-color-white;
+    width: 100%;
+}
+
+.container {
+    max-width: 50rem; // should match ReviewSubmit max-width
```

Comment:
Since we are adding a second instance of this and it relates to a third variable in ReviewSubmit this is a good chance to pull out into global css variable in `custom.scss`.  Something like `width-max-contained`  or whatever you like to name. We aren't consisted with our naming or casing in scss css fyi.